src/api/Base.js

```diff
@@ -180,6 +181,16 @@ class Base {
         });
     }
 
+    /**
+     * Formats the api response to usable data
+     * @param {Object} response the api response data
+     * @return {Object} the formatted api response data
+     */
+    formatResponse(response: Object): Object {
+        // Default returns unmodified response
+        return response;
```

Comment:
Let's look into using responseInterceptors possibly

Reply:
One way to avoid having another inheritance-function, is to override the successHandler itself in the chidlren.
Duplication of the one-line isDestroyed will happen though, but maybe thats fine.

Reply:
`        if (!this.isDestroyed() && typeof this.successCallback === 'function') {`
Looks like we're checking this in base already. Do we need need this elsewhere?

Reply:
@pramodsum I can use the same logic to format response data from a successful comment/task creation, possibly more if the other edit/delete endpoints return the same object structure.

Reply:
👍